src/components/ec-option-card/types.ts

```diff
@@ -0,0 +1,29 @@
+import type { IconName } from '../ec-icon/types';
+
+export interface OptionCardProps {
+  isDisabled?: boolean,
+  isSubmit?: boolean,
```

Comment:
I can't see this prop in vue file, what it is for? The interfaces should be the same.

Reply:
sorry forgot to remove it